packages/diagrams/lib/src/card.dart

```diff
@@ -35,20 +35,17 @@ class CardDiagram extends StatelessWidget implements DiagramMetadata {
                 title: Text('The Enchanted Nightingale'),
                 subtitle: Text('Music by Julie Gable. Lyrics by Sidney Stein.'),
               ),
-              new ButtonTheme.bar(
-                // make buttons use the appropriate styles for cards
-                child: new ButtonBar(
-                  children: <Widget>[
-                    new FlatButton(
-                      child: const Text('BUY TICKETS'),
-                      onPressed: () {},
-                    ),
-                    new FlatButton(
-                      child: const Text('LISTEN'),
-                      onPressed: () {},
-                    ),
-                  ],
-                ),
+              new ButtonBar(
```

Comment:
SGTM. :) 
Does this diagram need to be re-generated?

Reply:
I don't believe so, since the functionality should be the same, but I'll go ahead and do so anyway just to be sure!

Reply:
Note: Regenerated to test and was the same as before!